aries-site/src/examples/components/header/HeaderActionExample.js

```diff
@@ -6,11 +6,21 @@ export const HeaderActionExample = () => {
   return (
     <Header fill="horizontal">
       <Button plain>
-        <Box direction="row" align="center" gap="medium">
+        <Box
+          direction="row"
+          align="center"
+          gap="medium"
+          // pad maintains accessible hit target
+          // non-responsive maintains same dimensions for mobile
+          pad={{ vertical: 'small' }}
+          responsive={false}
+        >
           <Hpe color="brand" />
           <Box direction="row" gap="xsmall">
```

Comment:
Made a comment regarding wrapping of the App Name, suggesting we investigate whether `<Text wordBreak="keep-all" ... />` or similar might present the App Name better at different viewport widths. If we decide to do that investigation (and it works well), should we also implement in these examples?

Reply:
Yes, we would want to extend across all examples if we find a better implementation.